composer.json

```diff
@@ -36,7 +36,7 @@
   ],
   "require-dev": {
     "behat/behat": "^3.4",
-    "phpspec/phpspec": "^4.3",
+    "phpspec/phpspec": "dev-master",
```

Comment:
Do we really need this change?

Reply:
I don't think dev-master itself it's necessary. But it would be nice to allow more versions like "^4.3 || ^5.0 || ^6.0" but I haven't tested...

Reply:
You would be able to do so? Thank you 🙏